### ISSUE TITLE
Installation Error

### ISSUE THREAD

**Issue by zheesaker** [nvisionative/nvQuickSite#317]
Currently getting this series of messages during the installation process:

![image](https://user-images.githubusercontent.com/77459854/104635402-12aad700-565f-11eb-9220-3bf07e293c81.png)
![image](https://user-images.githubusercontent.com/77459854/104635479-32da9600-565f-11eb-9a87-93db057d98fe.png)
 
 With these setting configured:
 
![image](https://user-images.githubusercontent.com/77459854/104635562-4a198380-565f-11eb-8af0-1ce512529bce.png)

Is it necessary to be connected to a database to finish installation? Thank you.

**Comment by david-poindexter** [comment]
@zheesaker `(local)` may not work depending on your local SQL Server instances and their configuration.  You may want to try using the name of the instance just in case it is not configured to use `(local)`.

**Comment by malonzo-77** [comment]
Hi David, I have received the exact same error as above. However, I did try the above answer with no avail. I've uninstalled IIS Express and re installed it as well as my SQL Server Express 2012 and 2017.

**Comment by valadas** [comment]
I don't think this tool expects to work with IIS Express, I think you need full IIS working, right @david-poindexter  ?

If that is the case, you will need to go into "Turn Windows Features On and Off" in the start menu and add IIS, for Dnn after that you will probably need after IIS is installed to add the UrlRewrite IIS module which I believe is not installed by default.

**Comment by malonzo-77** [comment]
Hi Daniel,

I was able to get it working. Yes, unfortunately I had to uninstall and reinstall IIS Express. I did have to go into "Turn Windows Features On and Off" in the start IIS, then in Visual Studio go under Tools -> Options -> Projects and Solutions -> Web Projects - "Use the 64 bit version of IIS Express for websites and projects" and check that. Then that error went away and am able to run a local DNN instance!

THANKS!!!!
[cid:image001.png@01D710FD.26185C60]
